images/prow-tests/Dockerfile

```diff
@@ -29,17 +33,16 @@ RUN docker-credential-gcr configure-docker
 RUN apt-get install -y uuid-runtime  # for uuidgen
 RUN apt-get install -y rubygems  # for mdl
 
-# Install go at 1.12 same as how it's installed in kubekins-e2e image, reference code here:
+# Install go at 1.13 same as how it's installed in kubekins-e2e image, reference code here:
 # https://github.com/kubernetes/test-infra/blob/1e9b5dc3de4b268aab57c9c48120aa3dcf096bc6/images/kubekins-e2e/Dockerfile#L64
-ENV GO_TARBALL "go1.12.linux-amd64.tar.gz"
+ENV GO_TARBALL "go1.13.linux-amd64.tar.gz"
 RUN rm -rf /usr/local/go && \
     wget -q "https://storage.googleapis.com/golang/${GO_TARBALL}" && \
     tar xzf "${GO_TARBALL}" -C /usr/local && \
     rm "${GO_TARBALL}"
 
 # Extra tools through go get
-RUN GO111MODULE=on go get github.com/google/ko/cmd/ko
-RUN GO111MODULE=on go get github.com/boz/kail/cmd/kail
+RUN go get -u github.com/google/ko/cmd/ko
```

Comment:
This is wrong, please rebase your PR: remove this line and keep the to `RUN` above for `ko` and `kail` with `GO111MODULE=on`.

Reply:
fixed, thanks